src/Compilers/CSharp/Portable/Symbols/Tuples/TupleErrorFieldSymbol.cs

```diff
@@ -0,0 +1,122 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// Represents a field of a tuple type (such as (int, byte).Item1)
+    /// that doesn't have a corresponding backing field within the tuple underlying type.
+    /// Created in response to an error condition.
```

Comment:
Does this type get instantiated for `(int, byte).Item1`?  If so then I find the comment confusing because it has a real backing field: `ValueTuple<int,byte>.Item1`. 

---
In reply to: [61766119](https://github.com/dotnet/roslyn/pull/11006#discussion_r61766119) [](ancestors = 61766119)